lib/PuppeteerSharp/CookieParam.cs

```diff
@@ -14,61 +14,59 @@ public class CookieParam
         /// Gets or sets the name.
         /// </summary>
         /// <value>The name.</value>
-        [JsonProperty("name")]
         public string Name { get; set; }
         /// <summary>
         /// Gets or sets the value.
         /// </summary>
         /// <value>The value.</value>
-        [JsonProperty("value")]
         public string Value { get; set; }
         /// <summary>
         /// Gets or sets the domain.
         /// </summary>
         /// <value>The domain.</value>
-        [JsonProperty("domain", NullValueHandling = NullValueHandling.Ignore)]
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
maybe use the `NullValueHandling = NullValueHandling.Ignore` in the json-settings instead?